domain/schema_validator.go

```diff
@@ -149,11 +149,10 @@ func (s *schemaValidator) Update(leaseGrantTS uint64, oldVer, currVer int64, cha
 			actionTypes = change.ActionTypes
 		}
 		for idx, ac := range actionTypes {
-			// NOTE: ac is not an action type, it is (1 << action type).
-			if ac == 1<<model.ActionUnlockTable {
+			if ac == uint64(model.ActionUnlockTable) {
 				s.do.Store().GetMemCache().Delete(tblIDs[idx])
 			}
-			if ac == 1<<model.ActionFlashbackCluster {
+			if ac == uint64(model.ActionFlashbackCluster) {
```

Comment:
Do we need to update https://github.com/pingcap/tidb/blob/master/session/schema_amender.go#L46 and https://github.com/pingcap/tidb/blob/master/session/schema_amender.go#L666 like this?

Reply:
updated, PTAL @zimulala